### DISCUSSION TITLE
Jarvis 3.1.8 - Import von Homematic Devices meldet "... undefinierte Datenpunkten (bitte dem Entwickler melden)"

### DISCUSSION THREAD

**Discussion by Carbon1974** [Zefau/ioBroker.jarvis#2472] [Ideas / Feature Requests]
Auf einem Raspberry Pi 4 - 4GB habe ich eine IOBroker (v.6.3.5) Dockerinstanz am Laufen. Die Anbindung von 5 Zigbee-Devices funktioniert problemlos. Die Verbindung zu einer bestehenden (realen) CCU2 von Homematic funktioniert denke ich ebenfalls. Über eine RPC-Instanz (v 1.15.19) sind einige wenige Homematic-IP Geräte angebunden und über eine zweite RPC-Instanz (ebenfalls 1.15.19) sind die übrigen Homematic Geräte angebunden.
Über eine Homematic Rega-Instanz (v 3.0.47) kann ich die Homematic Geräte auch steuern.

Die Homematic-Geräte möchte ich nun gerne via Jarvis viualisieren und steuern. Hierzu wollte ich die Devices in Jarvis einlesen und folgende Meldungen erhalten, die mir mitgeteilt haben, dass Geräte mit undefinierten Datenpunkten vorliegen und ich dies dem Entwickler melden solle :

![Import1](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/894e59c1-cd0a-4635-a4ca-3102ec617237)

![Import2](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/eaf72f39-a0a8-4d71-b034-6d6db7af2cdc)

![Import3](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/ed14699a-cb95-4fed-9681-dedb3bcef295)

![Import4](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/528c4839-8c92-4fa7-b975-5ff31d1345bb)

Da ich nicht genau weiß weche Informationen noch zum Verständnis notwendig sind, habe ich hier noch eine Übersicht der Objekte aus dem IOBroker eingefügt, die meine Homematic Devices betreffen.

Die erste Übersicht betrifft die RPC-Instanz 1 mit den Homematic-IP Geräten:

![HMIP 01](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/85cd3686-41a2-43dc-9559-084e933385d0)

![HMIP 02](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/af384b14-347e-4189-aee3-5de3a42b8935)

![HMIP 03](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/95f99eaf-f269-42d6-b260-fed8dc02dcbb)

![HMIP 04](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/cb8812d2-d4d6-4d07-8b5b-c329284df255)

![HMIP 05](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/e077f20d-57b2-4bf5-bc41-5f718563fe4d)

![HMIP 06](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/14d9659b-8be1-4e25-930e-ea68c330e150)

Die zweite Übersicht betrifft die RPC-Instanz 2 mit den Homematic (non-IP) Geräten

![HM NonIP 01](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/c09dc0b4-483f-4ae1-960c-0673ffea3a28)

![HM NonIP 02](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/7c4c17fe-0d6c-4481-a082-6309003cbf9f)

![HM NonIP 03](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/78b2f019-75df-4617-872d-853a693db64c)

![HM NonIP 04](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/e752db94-e2b9-4c32-8fe6-981421900ba8)

![HM NonIP 05](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/9d2eacd1-c983-4c9f-974e-03792a08434c)

![HM NonIP 06](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/3a055cf4-f555-495d-8359-5a8167d4c804)

![HM NonIP 07](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/3d1c5089-3935-4d3f-917f-567d4801270c)

![HM NonIP 08](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/d3189f8f-869d-45c4-a97d-5752adf065ac)

![HM NonIP 09](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/e152c27b-e5b2-4bf2-ac0b-f3f0c26b815e)

![HM NonIP 10](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/4eee1737-62ea-461c-beb5-82ec8eecfb1f)

![HM NonIP 11](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/2213e0a5-7d0a-4d00-9bde-ac85f46947e0)

![HM NonIP 12](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/259139a8-67a0-4fcd-b28d-0cdeff4d0989)

![HM NonIP 13](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/fb34c415-e197-4019-8f42-679f9c3f3c96)

![HM NonIP 14](https://github.com/Zefau/ioBroker.jarvis/assets/109553486/6a24648c-5593-4ffb-90c9-14006ac0cbd3)

 Über Feedback, Vorschläge, Anregungen wie ich zum Einbinden der Devices verfahren kann, würde ich mich sehr freuen.
 Wenn hierzu weitere Infos notwendig sind, liefere ich diese natürlich sehr gerne jederzeit.

**Comment by franky218** [comment]
Habe das gleiche Problem.
Allerdings primär bei "neuen" Geräten, wie z.B.:
- HmIP-SWDO-2 (Tür/Fenster-Kontakt); HmIP-SWDO funktioniert
- HmIP-WTH-B-2 (Raumthermostat)
Auch ich würde mich über einen Lösung freuen. Falls Datenpunkte benötigt werden, kann ich die natürlich gerne liefern.

LG Franky218

**Comment by Zefau** [comment]
@Carbon1974 @franky218 könnt ihr mir die JSON Exporte der jeweiligen Bäume zur Verfügung stellen?
Alternativ gingen auch die Datenpunkte, die ihr importiert haben möchtet.

Folgende Geräte sollten bereits verfügbar sein (bitte ggf. v3.2 prüfen):
- `HmIP-FROLL`
- `HmIP-FSM`
- `HmIP-FSM`
- `HmIP-SWDO-2`
- `HM-LC-Sw2-FM`
- `HM-LC-Bl1-FM` (Funk-Rolladenaktor)
- `HM-LC-Bl1PBU-FM` (Funk-Rolladenaktor)
- `HM-WDS100-C6-O-2`
- `HM-Sec-SD-2` sowie `HM-Sec-SD-2-Team`

`HmIP-RCV-50 HmIP-RCV-1` ist die Zentrale und nicht notwendig, oder?
`HmIP-WTH-B` habe ich drin und nehme ich als Vorlage für `HmIP-WTH-B-2`, der damit mit der nächsten v3.2-beta verfügbar sein wird.

Fehlen effektiv nur
- `Fernbedienung` ist welches Homematic IP Gerät?
- `HM-LC-Sw4-Ba-PCB`
- `HM-Sen-RD-O`
